prow.go

```diff
@@ -130,12 +130,26 @@ func (r *URLConfigResolver) Resolve(org, repo, branch, variant string) ([]byte,
 	}
 }
 
-// stopJob triggers namespace deletion, which will cause graceful shutdown of the
-// cluster. If this method returns nil, it is safe to consider the cluster released.
+// stopJob triggers graceful cluster teardown. If this method returns nil,
+// it is safe to consider the cluster released.
 func (m *jobManager) stopJob(name string) error {
```

Comment:
In both cases you should just delete the Pod running `ci-operator` in the appropriate `ci` namespace

Reply:
Thanks @stevekuznetsov . Verified this worked for both templates and steps and dramatically simplifies things. Added a new commit with the change.

Reply:
After trying this at various points of the install, discovered https://github.com/openshift/release/pull/16552 where data written into SHARED_DIR is not available to the post pod. Will need to track this down before adopting this approach.

Reply:
Fixed with workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1937523: 
- https://github.com/openshift/ci-tools/pull/1805 
- https://github.com/openshift/ci-tools/pull/1823